tests/test_container.py

```diff
@@ -1155,6 +1155,13 @@ def test_get_objects_stream_closes(temp_container, generate_random_data):
        when it goes out of scope - so I add also the test that, inside the loop, at most one more file is open.
        The final check seems to always pass even if I forget to do close some file.
     """
+    current_process = psutil.Process()
+    # We note the number of open of files, since Windows by default has some files open independent of the container
+    begin_test_open_files = len(current_process.open_files())
+
+    temp_container = Container(temp_dir)
```

Comment:
It is fine for a test, but this means you didn't drop the container after, correct?

Reply:
Right should everything should be wrapped by a finally and close the container